Signal/src/Storyboards/Registration.storyboard

```diff
@@ -89,9 +86,6 @@
                                         <state key="normal" title="Country Code">
                                             <color key="titleColor" red="0.0" green="0.0" blue="0.0" alpha="1" colorSpace="custom" customColorSpace="sRGB"/>
                                         </state>
-                                        <connections>
-                                            <segue destination="CCL-Zs-hfU" kind="presentation" id="uxC-Up-OyM"/>
-                                        </connections>
```

Comment:
We now navigate between "registration" view and "country code" view programmatically without segues.